src/Pages/AboutUsManager/AboutUsManager.js

```diff
@@ -0,0 +1,113 @@
+import Header from '../../Components/Header/Header';
+import './AboutUsManager.css';
+export default function AboutUsManager(props){
+  const [modal, setModal] = useState(false);
+  const [officer, setOfficer] = useState();
+  const [modalState, setModalState] = useState(officerModalState.SUBMIT);
+  const [officers, setOfficers] = useState([]);
+  const headerProps = {
+    title: 'About Us Manager'
+  };
+
+
+  async function populateOfficers(){
+    const officerData = await getAllOfficers();
+    if(!officerData.error) setOfficers(officerData.responseData);
+  }
+
+  useEffect(() => {
+    populateOfficers();
+  }, []);
+
+  function toggle(){
+    setModal(!modal);
+  }
+
+  function toggleEditOfficer(officer) {
+    setModalState(officerModalState.EDIT);
+    setOfficer(officer);
+    toggle();
+  }
+
+  function toggleCreateOfficer(officer) {
+    setOfficer();
+    setModalState(officerModalState.SUBMIT);
+    setModal(!modal);
+  }
+
+  async function handleSubmit(officer) {
+    if (modalState === officerModalState.SUBMIT) {
+      await createOfficer(officer, props.user.token);
+    } else if(modalState === officerModalState.EDIT) {
+      await editOfficer(officer, props.user.token);
+    }
+  }
+
+  return (
+    <>
+      <Header {...headerProps} />
+      <main className="officer-body">
+        <section className="exec-container">
+          <Button onClick={toggleCreateOfficer} className="create-officer">
+                      Add Officer +
+          </Button>
+          {modal && (<AboutUsManagerModal
+            modal={modal}
+            toggle={toggle}
+            handleDelete={() => deleteOfficer(officer, props.user.token)}
+            handleSubmit={handleSubmit}
+            modalState={modalState}
+            populateOfficers={populateOfficers}
+            token={props.user.token}
+            {...officer}
+          />)}
+          <h1>Executive Team</h1>
+          <div className="grid-container">
+            {officers.length ? (officers.map((officer, i)=> {
+              if(officer.team === 'executive'){
+                return(
+                  <AboutUsCard
+                    handleClick={() => toggleEditOfficer(officer)}
+                    info={officer}
+                    key={i}/>
+                );
+              } return null;
+            })): (
+              <h1>No officers yet!</h1>
+            )}
+          </div>
+        </section>
+        <section className="officer-container">
+          <h1>Meet Our Team</h1>
+          <div className="grid-container-circle">
+            {officers.length ? (officers.map((officer, i) => {
+              if(officer.team === 'officers'){
+                return(
+                  <AboutUsCircle
+                    handleClick={() => toggleEditOfficer(officer)}
+                    info={officer}
+                    key={i}/>
+                );
+              } return null;
+            })): (
+              <h1>No officers yet!</h1>
```

Comment:
```suggestion
              <span>No officers yet!</span>
```

I think it's a bit weird that the containing header has the same level as the text.